README.md

```
@@ -82,103 +102,212 @@ $meineVorlage = '
+```
 
-### `mediaUrl(string $type, string $file)`
-Generiert korrekte URLs für Media-Manager-Bilder im PDF:
+**Best Practices für Zertifikate:**
+- **Produktive Zertifikate:** Nur von vertrauenswürdigen CAs verwenden
+- **Dateiberechtigungen:** 600 (nur Webserver lesbar) setzen
+- **Pfad-Sicherheit:** Zertifikate außerhalb des Web-Root speichern
+- **Ablaufmonitoring:** Rechtzeitige Erneuerung vor Ablauf
+- **Backup:** Sichere Aufbewahrung von Zertifikaten und Passwörtern
+        'Dokument-Freigabe',         // Grund der Signierung
+        'max@example.com'            // Kontaktinformationen (optional)
```

markdownlint-cli2 (0.17.2)

Bare URL used
null

(MD034, no-bare-urls)